.circleci/config.yml

```diff
@@ -31,115 +32,47 @@ parameters:
     type: string
     default: ''
 
-commands:
-  bootstrap_pipeline_circleci_user:
-    steps:
-      - run:
-          name: Initialize CIRCLE_USERNAME from pipeline
-          command: |
-            echo "export CIRCLE_USERNAME=<< pipeline.parameters.circleci-user >>" >> $BASH_ENV
-            source $BASH_ENV
-
-jobs:
-  integration-test-notify:
-    <<: *default_build_environment
-    steps:
-      - checkout
-      - utils/notify:
-          on_success: true
-
-  integration-test-slack:
-    <<: *default_build_environment
-    steps:
-      - checkout
-      - bootstrap_pipeline_circleci_user
-      - utils/slack-notify-waiting-for-approval:
-          slack_bot_token: ${SLACK_ACCESS_TOKEN}
-
-  integration-test-cancel-older-jobs:
-    <<: *default_build_environment
-    steps:
-      - checkout
-      - utils/cancel-older-awaiting-approvals
 
 workflows:
-  test-pack:
-    unless: << pipeline.parameters.run-integration-tests >>
+  lint-pack:
     jobs:
-      - orb-tools/pack:
-          context:
-            - dockerhub
+      - orb-tools/pack
       - shellcheck/check:
-          context:
-            - dockerhub
-          dir: ./src/scripts
-      - bats/run:
-          context:
-            - dockerhub
-            - slack
-          path: ./src/tests
-      - hold-for-dev-publish:
-          context:
+          exclude: 'SC2148,SC2038,SC2086,SC2002,SC2016'
```

Comment:
Can you add comments about what these are?  Or perhaps better? than details about the specific errors is a link to all of the errors.  Also, are these ones that we want to permanently keep or is this just the current state?  Some idea of changes that we should make in the future, if any, would be good.